soccer/gameplay/plays/restarts/basic_corner_kick.py

```diff
@@ -0,0 +1,60 @@
+import main
+import robocup
+import behavior
+import constants
+
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+
+#Holds a point or a line, with the associted priority(0 is highest priority)
+class PointAndPriority:
+	def __init__(self, line_pt, priority):
+		self.line_pt = line_pt
+		self.priority = priority
+
+class BasicCorner(standard_play.StandardPlay):
+	def __init__(self):
+		super().__init__(continuous = True)
+		#the minimum number we want the probability multiplier be
+		MINIMUM_MULTIPLIER = 0.5
```

Comment:
Could you go into a little more depth on what this config value represents? It sounds as if it's the maximum penalty on pass chance between the highest priority and lowest priority segment, but I'm not sure